inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterNodeBindLabelRequest.java

```diff
@@ -29,11 +29,11 @@
  */
 @Data
 @ApiModel("Cluster node bind and unbind tag request")
-public class ClusterNodeBindTagRequest {
+public class ClusterNodeBindLabelRequest {
 
     @NotBlank(message = "Cluster nodeTag cannot be blank")
     @ApiModelProperty(value = "Cluster node tag")
```

Comment:
Add explanation above. like `@ApiModelProperty(value = "MQ resource", notes = "in inlong group, TubeMQ corresponds to Topic, Pulsar corresponds to Namespace")`